frontend/src/controller/Fetching.ts

```diff
@@ -22,85 +22,101 @@ export const postLogin = (loginData: LoginData) =>
 // }
 
 export async function getPing() {
-    const rawResponse = await fetch(`/service/ping`, {
+    return await fetch(`/service/ping`, {
         method: 'GET',
         headers: {"Authorization": "Bearer" + localStorage.getItem(STORAGE_KEY_TOKEN) || "No Token"},
     });
-    return rawResponse
-};
+}
+
+function handelError(response: Response):Response {
+    if (response.status === 403) {
+        window.location.reload();
+    }
+    return response;
+}
+
 
 export async function postSingleFader(faderItem: FaderItem) {
-    await fetch(`/api/faderpage/setvalue`, {
+
+    handelError(await fetch(`/api/faderpage/setvalue`, {
         method: 'POST',
         headers: {"Authorization": "Bearer" + localStorage.getItem(STORAGE_KEY_TOKEN) || "kein Token",'Content-Type': 'application/json'},
```

Comment:
I would not store the token in the localsStorage. I would create a state and maybe a context for authorisation stuff. If you want to still be signed in after a reload you could save the token in the localstorage when logging in and try to load it when you open your site again.